lib/Controller/ContactController.php

```diff
@@ -173,32 +178,32 @@ public function searchAttendee(string $search):JSONResponse {
 	 * @param string $circleId CircleId to query for members
 	 * @return JSONResponse
 	 * @throws Exception
-	 * @throws \OCP\AppFramework\QueryException
 	 *
 	 * @NoAdminRequired
 	 */
 	public function getCircleMembers(string $circleId):JSONResponse {
-		if (!$this->appManager->isEnabledForUser('circles') || !class_exists('\OCA\Circles\Api\v1\Circles')) {
+		if (!class_exists('\OCA\Circles\Api\v1\Circles') || !$this->appManager->isEnabledForUser('circles')) {
+			$this->logger->debug('Circles not enabled');
```

Comment:
Why the debug log?

Reply:
Why not?

Reply:
The method is only called, from the frontend, when circles is enabled.

Reply:
apparently not otherwise we wouldn't need the class_exists check / app enabled check